docs/build/dev-workflow/personas.md

```diff
@@ -11,13 +11,13 @@ The aim is to provide an overview of the main technical user personas, how they
 
 ## Polymer, Ethereum's IBC hub
 
-Polymer is bringing IBC to Ethereum rollups. [IBC](../../learn/concepts/ibc/ibc.md) or the Inter-Blockchain communication protocol is a layered protocol that does a good job of separating the concerns of application developers, and core developers (consisting of transport and state layers).
+Polymer is bringing IBC to Ethereum rollups. [IBC](../../learn/concepts/ibc/ibc.md) or the _Inter-Blockchain communication protocol_ is a layered protocol that does a good job of separating the concerns of application developers, and core developers (consisting of transport and state layers).
 
 Polymer as an IBC Hub takes care of the IBC transport layer, providing a straightforward and permissionless integration path, starting with OP stack rollups. Rollup chain devs or rollup framework devs are then potential users to integrate their rollups or frameworks with Polymer.
 
 Not only on the application layer, we expect a variety of builders to build. Also on the state layer, we expect a thriving market in IBC clients to arise by developers specializing in state verification along the security/latency/cost trade-off space.
 
-As Polymer grows, it will become more decentralized and infra operators who will run relayers and verifier nodes for Polymer will also make up 
+As Polymer grows, it will become more decentralized and infra operators -who will run relayers and verifier nodes for Polymer- will also be part of Polymer's ecosystem of users.
```

Comment:
This sentence can be simplified. 

As Polymer grows, infrastructure operators will join the network as relayers and verification nodes, which will progressively decentralized the network. 

[recommendation] avoid shortened phrases in docs such as "infra"

Reply:
Adjusted along with next comment. Do think the term "infra operator" has become significantly ingrained into the industry to consider it official jargon but nevertheless updated it 👍